src/css/flags/ads.less

```diff
@@ -55,6 +55,14 @@
         }
     }
 
+    .jw-overlays {
+        .jw-plugin {
+            video {
+                cursor: pointer;
+            }
+        }
+    }
```

Comment:
It will get compiled that way. Since we're using less we write our rules using a hierarchy. It make it easier to read and modify.